src/rpc/channels.ts

```diff
@@ -23,6 +23,16 @@ export interface Channel extends EventEmitter {
 }
 
 
+export interface PlaywrightChannel extends Channel {
+}
+export type PlaywrightInitializer = {
+  chromium: BrowserTypeChannel,
+  firefox: BrowserTypeChannel,
+  webkit: BrowserTypeChannel,
+  deviceDescriptors: types.Devices,
```

Comment:
Are we not going to use this in javascript client, because it is not available synchronously? If so, why do we need it?

Reply:
It is a reference implementation.